components/component-stylesheets/alerts.css

```diff
@@ -0,0 +1,39 @@
+
+.alert{
+    font-size: 1.5rem;
+    padding: 1rem 1rem;
```

Comment:
you can use the shorthand property here, if you want to give 1rem to top-bottom & left-right.

Reply:
thanks for pointing it out. fixed in recent commit.